code/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
+import './components/thirdquestion.css'
```

Comment:
I imported the separate css files for the components into the respective component. Interesting to see that you can also do it this way!